### PR TITLE
QuickId: Update designs

### PR DESCRIPTION
**Pull request by davidyuk**
screenshot from 2018-09-18 06-29-34

### CODE REVIEW

**Comment by davidyuk** [comment]
screenshot from 2018-10-10 10-09-57
It is ready, except for the lack of `browser` and `wallet` icons.

**Comment by davidyuk** [comment]
screenshot from 2018-10-11 15-11-30

File: src/components/FooterMobile.vue [hunk 1]
```diff
@@ -20,20 +19,15 @@
...
 import ApproveMessage from './ApproveMessage.vue';
 import ApproveTransaction from './ApproveTransaction.vue';
 
 export default {
   components: {
-    QuickId, Accounts, ApproveMessage, ApproveTransaction,
-  },
-  props: {
-    showBackButton: { type: Boolean, default: false },
+    QuickId, ApproveMessage, ApproveTransaction,
```
Comment (sadiqevani) [inline src/components/FooterMobile.vue]: @davidyuk 

Isn't it better if it is like:

```
{
  QuickId, 
  ApproveMessage, 
  ApproveTransaction,
}
```

Comment (davidyuk) [reply src/components/FooterMobile.vue]: I'm not sure, this style is allowed by current linter settings. If a developer adds additional components into this line, linter will ask to write as you proposed, but now it is not necessary.

**Pull request merged**